packages/console/src/pages/Organizations/PermissionsField/index.tsx

```diff
@@ -32,6 +33,7 @@ function PermissionsField() {
   const api = useApi();
   const [isModalOpen, setIsModalOpen] = useState(false);
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
+  const [editData, setEditData] = useState<Nullable<OrganizationScope>>(null);
```

Comment:
Why `Nullable` instead of `Optional`, any specific reasons?

Reply:
use this to force the `editData` prop to be present when using the component